README.txt

```diff
@@ -1,84 +1,90 @@
 === Event Codes - Shortcodes that work with other event plugins ===
 Contributors: WPdwarves
-Tags: event, events, calendar, event code, codes, shortcode, modern tribe
+Tags: event shortcode, event calendar shortcode, shortcodes, event, events, calendar, modern tribe
 Requires at least: 4.1
 Tested up to: 4.7.4
 Stable tag: 1.0.0

```

Comment:
Need to update version to 1.0.1 everywhere, find 1.0.0 and replace, there will be 2 - 3 places in code that need this increment